Frends.FixedWidthFlatFile.ConvertToJSON/Frends.FixedWidthFlatFile.ConvertToJSON/FixedWidthFlatFile.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Frends.FixedWidthFlatFile.ConvertToJSON.Definitions;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+
+#pragma warning disable 1591
+
+namespace Frends.FixedWidthFlatFile.ConvertToJSON
+{
+    public static class FixedWidthFlatFile
+    {
+        /// <summary>
+        /// This is a task that converts given value to json string.
+        /// Documentation: https://github.com/CommunityHiQ/Frends.FixedWidthFlatFile.ConvertToJSON
```

Comment:
Please change the documentation link to `[Documentation](link)`, so it will be an embedded link in the Frends UI, like this: [Documentation](link). Also, the link should point to task portal URL according to latest guidelines: https://github.com/FrendsPlatform/FrendsTasks/wiki#task-checklist.